cmd/service-watcher/main.go

```diff
@@ -126,14 +160,32 @@ func runMain() error {
 
 			log.Printf("Service failed. unitName: %s, activeState: %s", unitName, activeState)
 			for _, l := range rawLogs {
-				log.Println(l)
+				log.Debug(l)
+			}
+
+			version := "unknown"
+			packageName, ok := serviceToPackageMap[unitName]
+			if ok {
+				version, err = getPackageVersion(packageName)
+				if err != nil {
+					log.Printf("failed to get version for package %s: %v", packageName, err)
+				}
+			} else {
+				log.Infof("Unknown unitName: %s", unitName)
+			}
+			log.Debug("Version: ", version)
+
+			// If it is a snapshot then we don't need to be making service errors.
+			if strings.Contains(version, "SNAPSHOT") {
+				log.Infof("Skipping making service error for SNAPSHOT. Unit '%s', version '%s'", unitName, version)
+				break
 			}
 
 			event := eventclient.Event{
 				Timestamp: ts,
 				Type:      "systemError",
 				Details: map[string]interface{}{
-					"version":     1,
```

Comment:
I don't know why we were setting the version to 1. Maybe that was a placeholder for setting the package version but just wasn't added until now.